src/utils.sh

```diff
@@ -440,3 +453,41 @@ function handlers_init() {
 }
 
 export -f handlers_init
+
+# Constants
+InstrumentationKey="e83c2a5c-35d1-4a37-a5fe-b4f0f6e8e2a5"
+IngestionEndpoint="https://dc.services.visualstudio.com/v2/track"
+EventName="Azure-IoT-Edge-Installer-Summary"
+DeviceUniqueID="xinzedPC"
+CurrentTime=$(echo `date '+%Y-%m-%dT%H:%M:%S.%N'`)
+SchemaVersion="1.0"
+
+######################################
+# send_appinsight_telemetry
```

Comment:
>send_appinsight_telemetry [](http://example.com/codeflow?start=2&length=25)

nit on the naming, i suggest send_appinsights_event_telemetry. There are many different AI objects that can be sent but this function is sending - "baseType": "EventData"

Reply:
addressed in new commit